products/android_studio.md

```diff
@@ -0,0 +1,79 @@
+The particular example in that section discusses configuring a plugin project for PhpStorm, so the details for an Android Studio plugin project are reviewed here.
+
+Here are the steps to configure the `build.gradle` file for developing a plugin to target Android Studio:
+* The Gradle plugin attributes describing the configuration of the [IntelliJ Platform used to build the plugin project](/tutorials/build_system/gradle_guide.md#configuring-the-gradle-plugin-for-building-intellij-platform-plugin-projects) must be explicitly set. 
+  Continuing with the example [above](#matching-versions-of-the-intellij-platform-with-the-android-studio-version), set the `intellij.version` value to `191.8026.42`.
+* Android Studio plugin projects that use APIs from the `android` plugin must declare a dependency on that plugin.
+ Declare the dependency in `build.gradle` using the Gradle plugin `intellij.plugins` attribute, which in this case lists the [directory name](https://github.com/JetBrains/gradle-intellij-plugin/blob/master/README.md#intellij-platform-properties) of the plugin.
+* The best practice is to use the target version of Android Studio as the IDE Development Instance.
+  Set the Development Instance to the (user-specific) absolute path to the target Android Studio application.
+
+The snippet below is an example of configuring the Setup and Running DSLs in a `build.gradle` specific to developing a plugin targeted at Android Studio.
+```groovy
+  intellij {
+    // Define IntelliJ Platform against which to build the plugin project.
+    version '191.8026.42'  // Same IntelliJ IDEA version (2019.1.4) as target v3.5 Android Studio   
+    type 'IC'              // Use IntelliJ IDEA CE because it's the basis of the IntelliJ Platform   
+    // Require the Android plugin, Gradle will match the plugin version to intellij.version 
+    plugins 'android'     
+  }
+  
+  runIde {
+      // Absolute path to installed target v3.5 Android Studio to use as IDE Development Instance
+      ideaDirectory '/Applications/apps/AndroidStudio/ch-0/191.5791312/Android Studio.app'
+  }
+```
+
+### Configuring the Plugin plugin.xml File
+As discussed in the [Plugin Dependencies](/basics/getting_started/plugin_compatibility.md#declaring-plugin-dependencies) section of this guide, a plugin's dependency on [Modules Specific to Functionality](/basics/getting_started/plugin_compatibility.md#modules-specific-to-functionality) must be declared in `plugin.xml`. 
+When using Android Studio-specific features (APIs), a dependency on `com.intellij.modules.androidstudio` must be declared as shown in the code snippet below.
+Otherwise, if only general, platform features (APIs) are used, then a dependency on `com.intellij.modules.platform` must be declared as discussed in [Plugin Compatibility with IntelliJ Platform Products](/basics/getting_started/plugin_compatibility.md).
```

Comment:
I'd take out the comma after "general" and reword slightly to specify it's the IJ platform you talk about

Reply:
Agreed.